### DISCUSSION TITLE
点击，如果你是Linux User

### DISCUSSION THREAD

**Discussion by inkluv** [share121/inter-knot#1647] [其他]
来源：Bilibili @绿猫GreenCat

讨论区说下你的操作系统呗awa

放点大家爱看的
<img width="260" alt="1" src="https://github.com/user-attachments/assets/74329176-a22a-4396-a51b-cead0414944c">
<img width="268" alt="2" src="https://github.com/user-attachments/assets/e6ad782d-dbf9-4c88-b275-f20d1fafeee9">
<img width="536" alt="3" src="https://github.com/user-attachments/assets/61381f37-b9da-434c-8ebc-766e4806ddb2">
<img width="614" alt="4" src="https://github.com/user-attachments/assets/2d0bab8c-7bbb-4bbe-909a-e93852425da0">
<img width="659" alt="11" src="https://github.com/user-attachments/assets/ac70aec9-bb17-4f5d-a3f9-117f80f3d615">
<img width="899" alt="12" src="https://github.com/user-attachments/assets/dba54efc-4633-4d2e-8f44-2da29e56b7f1">
<img width="803" alt="5" src="https://github.com/user-attachments/assets/c5722b29-b386-4511-bf6b-d1bce16a91a9">

**Comment by inkluv** [comment]
好吧虽然我不是Linux User

**Comment by Buer-Nahida** [comment]
我也是Linux(NixOS)

**Comment by inkluv** [comment]
sudo 成为传奇绳匠！

**Reply by Buer-Nahida** [reply]
sudo rm -rf /*

**Comment by Buer-Nahida** [comment]
给你来张

![8dd1ea00aec446ba4df4d06bab3b4023](https://github.com/user-attachments/assets/7f72859a-a619-4e1f-9c9d-00d827aca78c)

**Comment by inkluv** [comment]
好看爱看

**Comment by HDTianRu** [comment]
Android也算一种Linux罢（）

**Comment by blqdswkl** [comment]
N3450的焯鸡牛逼的CPU跑windows 11！！！还有谁！！！
